common/perception_utils/include/perception_utils/object_classification.hpp

```diff
@@ -23,72 +23,75 @@ namespace perception_utils
 {
 using autoware_auto_perception_msgs::msg::ObjectClassification;
 
-inline std::uint8_t getHighestProbLabel(
+inline ObjectClassification getHighestProbClassification(
   const std::vector<ObjectClassification> & object_classifications)
 {
-  std::uint8_t label = ObjectClassification::UNKNOWN;
-  float highest_prob = 0.0;
-  // TODO(Satoshi Tanaka): It might be simple if you use STL or range-v3.
-  for (const auto & object_classification : object_classifications) {
-    if (highest_prob < object_classification.probability) {
-      highest_prob = object_classification.probability;
-      label = object_classification.label;
-    }
+  if (object_classifications.empty()) {
+    return ObjectClassification{};
+  } else {
```

Comment:
It's recommended to remove redundant `else`.
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f56-avoid-unnecessary-condition-nesting

Reply:
fixed at [f340cff](https://github.com/autowarefoundation/autoware.universe/pull/2042/commits/f340cfffed07bbfaaefa9bfc2b47219128686422)